+Prefs/+Inputs/ButtonField.m

```diff
@@ -1,10 +1,6 @@
 classdef ButtonField < Prefs.Inputs.LabelControlBasic
```

Comment:
seems like this class doesn't do that much anymore. Is there an example of how it can be used?

Reply:
Adding this in code.

Reply:
Also, @mwalsh161 , the previous ButtonField code was renamed to FileField (it had file stuff, so it should be different).